src/Tizen.NUI/src/public/Theme/Theme.cs

```diff
@@ -107,6 +107,24 @@ public Theme(string xamlFile) : this()
         /// <since_tizen> 9 </since_tizen>
         public string Version { get; set; } = null;
 
+        /// <summary>
+        /// The url of small broken image
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
+        public string SmallBrokenImageUrl { get; set; } = null;
+
+        /// <summary>
+        /// The url of  broken image
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
+        public string BrokenImageUrl { get; set; } = null;
+
+        /// <summary>
+        /// The url of large broken image
+        /// </summary>
+        /// <since_tizen> 9 </since_tizen>
+        public string LargeBrokenImageUrl { get; set; } = null;
```

Comment:
Need to add as 'Hidden API'.
- LargeBrokenImageUrl
- BrokenImageUrl
- SmallBrokenImageUrl